sys-fs/squashfs_tools/patches/squashfs_tools-4.3-gcc2.patch

```diff
@@ -0,0 +1,50 @@
+--- a/unsquash-2.c
++++ b/unsquash-2.c
+@@ -117,11 +117,11 @@ int read_fragment_table_2(long long *directory_table_end)
+ 
+ void read_fragment_2(unsigned int fragment, long long *start_block, int *size)
+ {
+-	TRACE("read_fragment: reading fragment %d\n", fragment);
+-
+ 	squashfs_fragment_entry_2 *fragment_entry = &fragment_table[fragment];
+ 	*start_block = fragment_entry->start_block;
+ 	*size = fragment_entry->size;
++
++	TRACE("read_fragment: reading fragment %d\n", fragment);
```

Comment:
You are right, it does not make any difference because we are not building with TRACE enabled.
So yes, that's just to make things easier to change, should anyone wish to compile with TRACE enabled.